content/post/086-ipfs-in-opera-for-android.md

```diff
@@ -0,0 +1,98 @@
+
+This release is a huge leap forward for the IPFS project, with a couple of important milestones:
+
+* This is the first time IPFS is *enabled by default*, right out of the box, in a production release of a major web browser.
+* This is the first time IPFS is able to be used directly as an *addressable protocol* in a production release of a mainstream web browser. You can type ipfs:// in the address bar, and Opera for Android will load the provided content address!
+* This is the first *mobile* web browser that is shipping IPFS support in a production release, easing access to decentralized content on the devices used by most internet users around the world.
+
+This is not just a win for IPFS, but for the distributed web, peer-to-peer, and the idea of content addressability:
+
+*For the first time, a major web browser does not ask the user to specify which server the content should be fetched from.*
+
+This is a paradigmatic change in the network architecture of the web! This type of bold experimentation by companies like Opera advances our goal of truly putting users in control of their experience on the internet.
+
+## Using IPFS in Opera for Android
+
+IPFS support is enabled by default, so as a user you don't need to do anything to turn it on.
+
+![Screenshots of entering an ipfs:// address in Opera for Android, and loading it.](https://i.imgur.com/xcCQtrT.png)
+
+You can load an IPFS content address such as:
+
+>[ipfs://bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/](ipfs://bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/)
+
+If you click the link above in Opera for Android you'll see Wikipedia, served from IPFS!
+
+Here's what that same address looks like from an HTTP gateway, which will work in all browsers:
+
+> https://ipfs.io/ipfs/bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/
+
+## ENS and DNS for human-readable addresses
+
+As you can see, raw IPFS addresses can be quite long and provide few human-readable elements. While many people do not understand what HTTP URLs actually mean, the human readable parts of them have meaning and build trust for users (We're publishing [some research](https://github.com/ipfs/browser-design-guidelines) about this soon!).
+
+There's a lot more work to do in maturing [naming in IPFS and distributed web efforts generally](https://github.com/ipfs/notes/blob/master/OPEN_PROBLEMS/HUMAN_READABLE_NAMING.md), but some services have already launched efforts to bridge this gap.
+
+You can register *decentralized domain names* that map to IPFS addresses using services like [Unstoppable Domains](unstoppabledomains.com/) or the [Ethereum Name System (ENS)](https://medium.com/the-ethereum-name-service/ethdns-9d56298fa38a?source=rss----63df5ce4636f---4).
+
+You can also register and use any *DNS* domain name, and map it to IPFS addresses using services like [Terminal.co](https://terminal.co) or tools like [DNSLink](https://docs.ipfs.io/guides/concepts/dnslink/).
```

Comment:
We should add screenshots here of ENS and terminal.co!

Reply:
I want to keep it at a shout-out, and not broaden the story overly much. "Naming in IPFS" could definitely be a standalone post though - is a complex landscape.

Reply:
Ok. Thought was an image much more clearly conveys how easy this is and what to do if you’re interested, but not required.

Reply:
Lidel and I talked about a "for web developers" version of this post - would be a good place for deeper dive.